### PR TITLE
add simpler safer next  example and update template

### PR DESCRIPTION
**Pull request by Cerchie**
- updates next example
- updates the template with a --public login option

### CODE REVIEW

File: TEMPLATE.md [hunk 1]
```diff
@@ -6,7 +6,11 @@ This project builds a GraphQL API... <!-- ADD YOUR OWN -->
 
 ## Getting Started
 
-You'll need to create a [StepZen account](https://stepzen.com/signup) first. Once you've got that set up, [git clone](https://www.atlassian.com/git/tutorials/setting-up-a-repository/git-clone) this repository onto your machine and open the working directory:
+You'll need to create a [StepZen account](https://stepzen.com/signup) first. 
+
+Alternatively, you can login using `stepzen login --public` without signing up, but note that your API will be publicly accessible.
```
Comment (royderks) [inline TEMPLATE.md:11]: People can also just run `stepzen start` without `stepzen login --public` before right?

File: with-next/.env [hunk 2]
```diff
@@ -0,0 +1 @@
+UNSPLASH_ACCESS_KEY=xMkNraNxvHSUtKnNNSZy9YRGIXhyAqpKFKdZFZwnHY8
```
Comment (royderks) [inline with-next/.env:1]: Do we want to expose this key?

Comment (Cerchie) [reply with-next/.env:1]: nope-- disabled the app this belongs to.

File: TEMPLATE.md (hunk 1 above)
Comment (Cerchie) [reply TEMPLATE.md:11]: yes, like in rest2graphql. This flow makes what's going on in the background clearer though I think

File: with-next/pages/index.js [hunk 3]
```diff
@@ -1,59 +1,78 @@
...
-import HelloWorld from '../components/HelloWorld'
+import Image from 'next/image'
 import styles from '../styles/Home.module.css'
 
-export default function Home() {
+
+// This gets called on every request
+export async function getServerSideProps() {
+
+  let headers = {};
```
[the remark this answers is not in the document]
Comment (Cerchie) [reply with-next/pages/index.js:7]: done!

**Pull request merged**